src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -533,6 +569,33 @@ export function getHealForLifebindHeal(event: HealEvent): HealEvent | null {
   return GetRelatedEvents(event, LIFEBIND_HEAL)[0] as HealEvent;
 }
 
+export function getEchoTypeForLifebind(event: HealEvent): ECHO_TYPE {
+  if (!HasRelatedEvent(event, LIFEBIND) || event.targetID === event.sourceID) {
+    return ECHO_TYPE.NONE;
+  }
+  const lifebindApplyEvent = GetRelatedEvents(event, LIFEBIND)[0] as ApplyBuffEvent;
```

Comment:
Your LIFEBIND eventlink now includes refreshBuff events as well, unsure if that will cause this to miss events since you're explicitly casting to Apply here